src/Containers.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import Listing from '../lib/cockpit-components-listing.jsx';
+import cockpit from 'cockpit';
+
+import ContainerDetails from './ContainerDetails.js';
+const _ = cockpit.gettext;
+
+class Containers extends React.Component {
+    constructor(props) {
+	super(props);
+	console.log(props.containers);
+	this.renderRow = this.renderRow.bind(this);
+	this.getContainerState = this.getContainerState.bind(this);
+    }
+
+    navigateToContainer (container) {
+	cockpit.location.go([ container.ID ]);
+    }
+    //TODO
+    deleteContainer(event, container) {
+	return undefined;
+    }
+    //TODO
+    stopContainer(container) {
+	return undefined;
+    }
+    //TODO
+    startContainer(container) {
+	return undefined;
+    }
+
+    //TODO
+    restartContainer(container) {
+	return undefined;
+    }
+
+    getContainerState(state) {
+	if (state.Status)
+	    return state.Status;
```

Comment:
This is confusing. Why is it sometimes a free-form `Status` and sometimes a fixed `state.Paused`? This deserves a comment, or only using one of them if it is redundant.

Reply:
Copied from dockerUI. Unnecessary for varlinkAPI, I've deleted it.